lxc/import.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"os"
+
+	"github.com/lxc/lxd/shared/i18n"
+	"github.com/spf13/cobra"
+
+	cli "github.com/lxc/lxd/shared/cmd"
+)
+
+type cmdImport struct {
+	global *cmdGlobal
+}
+
+func (c *cmdImport) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = i18n.G("import [<remote>:] <backup file>")
+	cmd.Short = i18n.G("Import container backup")
+	cmd.Long = cli.FormatSection(i18n.G("Description"), i18n.G(
+		`Import backups of containers including their snapshots.`))
+	cmd.Example = cli.FormatSection("", i18n.G(
+		`lxc export u1 backup0.tar.xz
+    Create the backup.
```

Comment:
And that should really say `Create a container called u1 using backup0.tar.xz as the source` or something along those lines

Reply:
fixed